src/create_venv.sh

```diff
@@ -1,136 +1,257 @@
 #!/usr/bin/env bash
 
+# Copyright (c) YugaByte, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied.  See the License for the specific language governing permissions and limitations
+# under the License.
+#
 
-# Use SHA of requirements.txt as a comment in requirements_frozen.txt
-# 
+[[ "${_YB_CREATE_VENV_INCLUDED:=""}" == "yes" ]] && return 0
```

Comment:
The `_` prefix is just to indicate this variable is used only locally in this file despite being exported and global.  It is a convention I picked up awhile ago, 
I double quote strings in tests b/c that is what the google bash style guide says to do and I personally find it more readable https://google.github.io/styleguide/shellguide.html#testing-strings

Reply:
OK, let's use this then:
```suggestion
[[ "${_YB_CREATE_VENV_INCLUDED:-}" == "true" ]] && return 0
```
In a lot of scripts we already use the `${VAR_NAME:-}` convention for treating undefined variable as an empty string. I see no need to introduce the alternative notation `${VAR_NAME:=""}`. Or we can discuss team-wide and change the convention all of our Bash codebases.

```
# git grep ':-}'
src/yugabyte-bash-common.sh:  if [[ ! ${YB_NUM_CPUS:-} =~ ^[0-9]+$ ]]; then
src/yugabyte-bash-common.sh:  if [[ -n ${YB_PRETEND_WE_ARE_ON_GCP:-} ]] || \
src/yugabyte-bash-common.sh:  if [[ -z ${is_running_on_gcp_exit_code:-} ]]; then
src/yugabyte-bash-common.sh:  if [[ -n "${yb_fatal_quiet:-}" ]]; then
src/yugabyte-bash-common.sh:  if [[ ${yb_log_quiet:-} == "true" ]]; then
src/yugabyte-bash-common.sh:  if [[ ${yb_log_quiet:-} == "true" ]]; then
src/yugabyte-bash-common.sh:  if [[ ${yb_log_quiet:-} == "true" ]]; then
src/yugabyte-bash-common.sh:  if [[ ${yb_log_quiet:-} == "true" ]]; then
src/yugabyte-bash-common.sh:    if [[ -z ${!var_name:-} ]]; then
src/yugabyte-bash-common.sh:  if [[ -n ${JAVA_HOME:-} && $JAVA_HOME != "$new_java_home" ]]; then
src/yugabyte-bash-common.sh:  if [[ -n ${VIRTUAL_ENV:-} && -f "$VIRTUAL_ENV/bin/activate" ]]; then
```

Also, similarly, for boolean variables, we already use `true` and `false`, as well as `1` and `0`. Because this one is an internal variable, we could use either here (for user-specified YB_... variables we use `1` and `0` more frequently). But I see no need to introduce the new `yes` / `no` notation.